common/gateway-storage/migrations/20240910120000_generic_client_id.sql

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+CREATE TABLE clients (
+   id              INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+   client_type     TEXT NOT NULL CHECK(client_type IN ('entry mixnet', 'entry wireguard', 'exit wireguard'))
```

Comment:
for completion sake we're missing exit mixnet. also, can you change the typing slightly? I feel like having no spaces in SQL might backfire : )

Reply:
I'm guessing you mean to remove the white spaces. Sure, no problem, I'll also add `exit_mixnet`